packages/dashboard/src/components/TilePicker/index.js

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useUniqueIdentifier } from '@rainbow-modules/hooks';
+import RenderIf from 'react-rainbow-components/components/RenderIf';
+import RequiredAsterisk from 'react-rainbow-components/components/RequiredAsterisk';
+import StyledError from 'react-rainbow-components/components/Input/styled/errorText';
+import { Provider } from './context';
+import { StyledContainer, StyledLabel, StyledOptionsContainer } from './styled';
+
+/**
+ * TilePicker can be either radio buttons or checkboxes that are visually enhanced.
+ */
+export default function TilePicker(props) {
+    const {
+        style,
+        label,
+        required,
+        error,
+        id,
+        children,
+        value,
+        multiple,
+        className,
+        name,
+        onChange,
+    } = props;
+
+    const errorUnique = useUniqueIdentifier('error-message');
+    const errorId = error ? errorUnique : undefined;
+    const nameUnique = useUniqueIdentifier('tile-picker');
+    const groupNameId = name || nameUnique;
+
+    const handleChange = (optionName, isChecked) => {
+        if (multiple) {
+            if (Array.isArray(value)) {
+                const currentValue = isChecked
+                    ? [...value, optionName]
+                    : value.filter((item) => item !== optionName);
+
+                return onChange(currentValue);
+            }
+
+            const currentValue = isChecked ? [optionName] : [];
+            return onChange(currentValue);
+        }
+        return onChange(optionName);
+    };
+
+    const context = {
+        ariaDescribedby: errorId,
+        groupName: groupNameId,
+        onChange: handleChange,
+        value,
+        multiple,
+    };
+
+    return (
+        <StyledContainer id={id} className={className} style={style}>
+            <RenderIf isTrue={!!label}>
+                <StyledLabel>
+                    <RequiredAsterisk required={required} />
+                    {label}
+                </StyledLabel>
+            </RenderIf>
+            <StyledOptionsContainer>
+                <Provider value={context}>{children}</Provider>
+            </StyledOptionsContainer>
+            <RenderIf isTrue={!!error}>
```

Comment:
this logic `!!error` should be done inside of RenderIf, this way we don't have to double negate everytime which is an ugly thing to do :)